samples/cellular/nrf_cloud_multi_service/overlay_coap.conf

```diff
@@ -0,0 +1,73 @@
+#
+# Copyright (c) 2022 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+# General config
+CONFIG_FPU=y
+CONFIG_NEWLIB_LIBC_NANO=n
+
+# Rate of cloud interactions
+# These are faster than one normally would use in a low power device.
+# This is strictly for demo purposes.
+CONFIG_SENSOR_SAMPLE_INTERVAL_SECONDS=120
+CONFIG_LOCATION_TRACKING_SAMPLE_INTERVAL_SECONDS=360
+CONFIG_COAP_FOTA_JOB_CHECK_RATE_MINUTES=5
+
+# Logs
+CONFIG_MULTI_SERVICE_LOG_LEVEL_DBG=y
+CONFIG_LOG=y
+CONFIG_NET_LOG=y
+CONFIG_LOG_PRINTK=y
+CONFIG_COAP_LOG_LEVEL_INF=y
+CONFIG_LOCATION_LOG_LEVEL_DBG=y
+CONFIG_DATE_TIME_LOG_LEVEL_DBG=y
+CONFIG_NRF_CLOUD_COAP_LOG_LEVEL_DBG=y
```

Comment:
Not sure if you're aware you still have these here -- if you are that's fine, no need to remove them, just making sure

Reply:
I'll remove them